compliance/aws/disallowed_regions/aws_disallowed_regions.pt

```diff
@@ -1,10 +1,16 @@
 name "AWS Disallowed Regions"
 rs_pt_ver 20180301
 type "policy"
-short_description "Report on any instances that are in a disallowed region with the option to Terminate them. \n See the [README](https://github.com/flexera/policy_templates/tree/master/aws/disallowed_regions) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
-long_description "Version: 1.0"
+short_description "Check for instances that are in a disallowed region with the option to terminate them. \n See the [README](https://github.com/flexera/policy_templates/tree/master/aws/disallowed_regions) and [docs.rightscale.com/policies](https://docs.rightscale.com/policies/) to learn more."
```

Comment:
The link to the readme is wrong, it's missing the compliance part of the path.

Reply:
Link to readme has been updated